tests/testthat.R

```diff
@@ -1,8 +1,6 @@
 library(testthat)
 library(govstyle)
 library(ggplot2)
-library(png)
-library(visualTest)
 # Install visualTest to allow checking against a reference plot.
```

Comment:
`library(visualTest)` has been removed – do we still need the 'Install visualTest to allow...' comment as well?